docs/02-for-app-authors/02-requirements.md

```diff
@@ -28,6 +31,87 @@ Every repository must contain a manifest describing how to build the application
 
 This manifest may import other manifest files in the same repository. It's quite common to add the [shared-modules](shared-modules) repository as a submodule.
 
+#### Example manifest
+
+This assumes that upstream brings appdata, icons and desktop files. If not, you can add them yourself.
+
+<Tabs groupId="manifest-language" defaultValue="json" queryString>
+  <TabItem value="json" label="json" default>
+
+```json title="io.github.example.MyCoolApp.json"
+{
+  "app-id": "io.github.example.MyCoolApp",
```

Comment:
```suggestion
  "id": "io.github.example.MyCoolApp",
```
`app-id ` is [deprecated](https://github.com/flatpak/flatpak-builder/blob/67c892cd802b56e2a0c80fe22b0aaa08edf020e8/data/flatpak-manifest.schema.json#L1241C26-L1241C26)